docs/annexes/including-security-information-in-SPDX.md

```diff
@@ -0,0 +1,364 @@
+      "externalRefType": "vulnerabilityExploitabilityAssessment",
+      "locator": "https://github.com/oasis-tcs/csaf/blob/master/csaf_2.0/examples/csaf/csaf_vex/2022-evd-uc-01-a-001.json"
+    }
+  ]
+}
+```
+
+### G.1.2.2 Linking to a CSAF Advisory
+To reference a CSAF Advisory document, include the document locator as an external reference of type `securityAdvisory` on a Package Element.
+
+```json
+{
+  "@type": "Package",
+  "@id": "urn:spdx.dev:pkg-rh-open-shift",
+  "name": "Red Hat OpenShift Enterprise",
+  "packageVersion": "3.6",
+  "externalRefs": [
+    {
+      "@type": "ExternalRef",
+      "externalRefType": "securityAdvisory",
+      "locator": "https://github.com/oasis-tcs/csaf/blob/master/csaf_2.0/examples/csaf/rhsa-2019_1862.json"
+    }
+  ]
+}
+```
+
+
+## G.1.3 Linking to a CycloneDX
+
+To reference to [CycloneDX](https://cyclonedx.org) formatted security information applicable to a package you need to first create a Package Element. Then use an External Reference to link to the CycloneDX document which contains information about the package.
+
+```json
+{
+  "@type": "Package",
+  "@id": "urn:spdx.dev:pkg-stack-cors",
+  "name": "stack-cors",
+  "packageVersion": "1.3.0",
+  "externalRefs": [
+    {
+      "@type": "ExternalRef",
+      "externalRefType": "securityOther",
+      "locator": "https://raw.githubusercontent.com/CycloneDX/bom-examples/ed522d1f051c364e045b87c20665003a0c4ea777/SBOM/laravel-7.12.0/bom.json"
+    }
+  ]
+}
+```
+
+## G.1.4 Linking to an OSV
+
+To include a reference to [Open Source Vulnerability](https://github.com/google/osv) (OSV) formatted security information applicable to a package you need to first create a Package Element. Then use an External Reference to link to the OSV advisory.
+
+```json
+{
+  "@type": "Package",
+  "@id": "urn:spdx.dev:pkg-Django",
+  "name": "Django",
+  "packageVersion": "2.2",
+  "externalRefs": [
+    {
+      "@type": "ExternalRef",
+      "externalRefType": "securityAdvisory",
+      "locator": "https://github.com/github/advisory-database/blob/6b9d5bc96a62bb845ee71e4551a214eb1457e2c6/advisories/github-reviewed/2022/04/GHSA-2gwj-7jmv-h26r/GHSA-2gwj-7jmv-h26r.json"
+    }
+  ]
+}
+```
+
+
+## G.1.5 Linking to an OmniBOR (formerly known as GitBOM)
+
+To reference an [OmniBOR](https://omnibor.io/) (Universal Bill Of Receipts) formatted security information applicable to a package you must first create a Package Element. Then use an External Identifier to link to the OmniBOR document.
+**TODO: Is this gitoid correct? do we need a different example here?**
+
+```json
+{
+  "@type": "Package",
+  "@id": "urn:spdx.dev:pkg-example",
+  "name": "Example",
+  "packageVersion": "1.2.3",
+  "externalIdentifiers": [
+    {
+      "@type": "ExternalIdentifier",
+      "externalIdentifierType": "gitoid",
+      "identifier": "gitoid:blob:sha1:bcb99b819dadaebdf2c8f88d92ee9024c45f9df3"
+    }
+  ]
+}
+```
+
+## G.1.6 Linking to a vulnerability disclosure document
```

Comment:
What is the difference between this and an advisory? (Especially since the links point to advisory data)